graphix/sim/tensornet.py

```diff
@@ -13,7 +18,7 @@ class TensorNetworkBackend:
     Executes the measurement pattern using TN expression of graph states.
     """
 
-    def __init__(self, pattern, graph_prep="auto", **kwargs):
+    def __init__(self, pattern, backend="tensornetwork", graph_prep="auto", **kwargs):
```

Comment:
Why did you add the `backend` argument? it seems not called anywhere

Reply:
I forgot to remove this and deleted it.